server/app.js

```diff
@@ -57,13 +57,20 @@ app.get("/trees", async function (req, res) {
         )
         )
         `;
-      query = {
-        text: sql,
-        values: []
-      };
-      const r = await pool.query(query);
-      console.log("trees:", r.rows.length);
-      r.rows.forEach(e => treeIds.push(e.id) );
+    query = {
+      text: sql,
+      values: []
+    };
+    const r = await pool.query(query);
+    console.log("trees:", r.rows.length);
+    r.rows.forEach(e => treeIds.push(e.id) );
+    
+    /*
```

Comment:
Previous code would lead to problem when there aren't any trees under the organization ( defined by map_name). 
So here we must give an impossible tree list to filter out all trees to display on the map.